build-system/server/app-index/index.js

```diff
@@ -43,7 +42,7 @@ async function serveIndex({url}, res, next) {
     return next();
   }
 
-  const css = (await fs.readFileAsync(mainCssFile)).toString();
+  const css = (await fs.promises.readFile(mainCssFile)).toString();
```

Comment:
Why are you using `await fs.promises.[method]` here and elsewhere in this PR? All `fs` methods have an equivalent `fs.[method]Sync`, so you don't have to use await/promise...

Reply:
Fixed in https://github.com/ampproject/amphtml/pull/25822/commits/70267df1a1dc7a6277a1e7cd6c773711aea4ecad.

Reply:
For posterity, I'd like to note that `await`ing an async call allows the Node event loop to do other work while a potentially slow disk I/O takes place. Since this isn't exactly a busy server there's not a significant performance cost here, and for readability the sync methods are nice, but await/promise does have that benefit.

Reply:
I wrote this originally. No strong reason 🤷🏻‍♂️

Reply:
Agree, in this case, `readFileSync` seems like the right thing to do.